packages/client/src/__tests__/integration/happy/blog-env-cockroach/schema.prisma

```diff
@@ -0,0 +1,36 @@
+datasource db {
+  provider = "postgres"
```

Comment:
This looks like you're testing a cockroach connection string, but with a postgres provider? Shouldn't this have the cockroach provider and the preview flag?

Reply:
Fixed.